extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -148,6 +160,45 @@ export class IframeTransport {
   }
 
   /**
+   * Uses the Long Task API to create an observer for when 3p vendor frames
+   * take more than 50ms of continuous CPU time.
+   * Currently the only action in response to that is to log. There is a
+   * grace of LONG_TASK_REPORTING_THRESHOLD_ occurrences before logging begins.
+   * @VisibleForTesting
+   */
+  createLongTaskObserver() {
+    if (this.longTaskObserver_ || !this.ampWin_.PerformanceObserver) {
+      return;
+    }
+    this.longTaskObserver_ = new this.ampWin_.PerformanceObserver(entryList => {
```

Comment:
I think it's OK to create new performaceObserver for now as long as multiple instance of performanceObserver isn't a performance issue itself. `JankMeter` has its limitation now, for example only create performanceObserver with viewer support performance tracking.
If you confirm that performance is not an issue, and decide to go with existing code. Please add a TODO to indicate that we will consider merging this with `JankMeter`. 

Otherwise I think we can consider making making performanceObserver a service with an observable, and create only one for the ampdoc.

Reply:
TODO done, testing not done yet.

Reply:
Testing PerformanceObserver creation: Creating 100k instances takes about 200ms. I ran the following code several times and saw results ranging from 101ms to 233ms:
```
var d1 = new Date();
var performanceObservers = [];
for (var i=0; i<100000; ++i) {
  performanceObservers.push(new PerformanceObserver(function(entryList) {
    console.log(i);
  }));
}
var d2 = new Date();
console.log(d1);
console.log(d2);
console.log(d2 - d1);
```